subprojects/core/src/main/java/org/gradle/plugin/use/internal/InternalPluginRequest.java

```diff
@@ -30,13 +30,13 @@
     @Nullable
     Object getArtifact();
 
-    boolean isConfigured();
-
     boolean isApply();
 
     int getLineNumber();
 
     String getScriptDisplayName();
 
     String getDisplayName();
+
+    boolean isEnriched();
```

Comment:
This method is not necessary. The resolver shouldn't care whether a request was modified by a rule or not. It just uses whatever the request says.